app/blog/[slug]/page.tsx

```
@@ -95,11 +94,23 @@ export default async function BlogDetailPage({ params }: { params: Promise<{ slu
     </article>
   );
 }
+
+async function ArticleBody({ contentPromise }: { contentPromise: Promise<{ md: string; html: string }> }) {
+  const content = await contentPromise;
+  return (
+    <div
+      className="prose prose-lg max-w-none dark:prose-invert text-foreground prose-headings:text-foreground prose-p:text-foreground prose-strong:text-foreground prose-code:text-foreground prose-pre:bg-muted prose-pre:text-foreground"
+      dangerouslySetInnerHTML={{ __html: content.html }}
+    />
+  );
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)